bitcoin/src/blockdata/transaction.rs

```diff
@@ -106,15 +114,19 @@ impl OutPoint {
     /// The number of bytes that an outpoint contributes to the size of a transaction.
     const SIZE: usize = 32 + 4; // The serialized lengths of txid and vout.
 
+    /// The "null" `OutPoint`.
+    ///
+    /// This value is used for coinbase transactions because they don't have any previous outputs.
```

Comment:
The other two have a much nicer doc, I'd like it to be (mostly) copied here.

Reply:
Done